.github/workflows/trigger-sync-lockfiles.yml

```diff
@@ -0,0 +1,14 @@
+name: Trigger sync lockfiles workflow
+
+on:
+  push:
+    paths: [Cargo.lock]
+
+jobs:
+    trigger-sync-lockfiles:
+        name: Trigger sync lockfiles workflow
+        uses: eclipse-zenoh/ci/.github/workflows/sync-lockfiles.yml@main
+        secrets:
+            github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Correct me if I'm wrong: I believe the resulting workflow won't have sufficient permissions to enable auto-merge on dependent repos.

Reply:
Fixed in 265aa05